src/Ombi/Controllers/V2/SearchController.cs

```diff
@@ -395,6 +395,19 @@ public async Task<ActorCredits> GetMoviesByActor(int actorId)
         {
             return await _movieEngineV2.GetMoviesByActor(actorId, null);
         }
+        
+        /// <summary>
+        /// Returns all the tv shows that is by the actor id 
+        /// </summary>
+        /// <param name="actorId">TheMovieDb Actor ID</param>
+        /// <returns></returns>
+        [HttpGet("actor/{actorId}/tv")]
+        [ProducesResponseType(StatusCodes.Status200OK)]
+        [ProducesDefaultResponseType]
+        public async Task<ActorCredits> GetTvByActor(int actorId)
+        {
+            return await _tvEngineV2.GetTvByActor(actorId, null);
```

Comment:
Let's pass the API the current language?

Maybe just pass it as a query parameter? 

e.g. `actor/1/tv?lang=en`.
You can do something like

```
public async Task<ActorCredits> GetTvByActor([FromRoute] int actorId, [FromQuery] string lang)
```

Reply:
Good catch, I never noticed. I copy-pasted this from getMoviesByActor, which means the current page in stable releases displays movies in English only.. It's something I overlooked in my localization PR. 
Maybe it's best if I take this on in another PR, along with the movies part?

Reply:
Yeah I'm fine for you to do that

Reply:
@tidusjar Mind if I ask why not use DefaultLanguageCode() instead of a query parameter? It seems to be used everywhere else in Ombi for displaying discover cards and I don't see the use case for it. See https://github.com/sephrat/Ombi/blob/0c4bd10b0f002fd67b55a07247828a7f40682433/src/Ombi.Core/Engine/MovieSearchEngine.cs#L126 for example.

Reply:
Good point, `DefaultLanguageCode` reads from the user so it's fine then :)